content/monitors/notifications.md

```diff
@@ -27,98 +27,139 @@ right people get notified so the problem can be resolved as soon as possible.
 
-{{< img src="monitors/notifications/templateconditionalrecover.png" alt="template conditional recover" responsive="true" popup="true" >}}
+**Note**:  
+
+* The tag template variables available depend on the tag group selected in Step 1 of the monitor editor. The possible options automatically populate at the bottom of the "Use message template variables" help box in Step 3 of the editor.  
+
+* Template variable content is escaped by default. If your variable
+contains JSON or code that you would NOT like to be escaped, use triple braces instead of double braces (e.g. `{{{event.text}}}`).
+
+* See a complete list of contextual template variables available to your monitor by clicking the "Use message template variables" link or in the list of suggestions that appears when you type `{{` to begin a template variable name.
+
+* The tag template variables can also be used in the monitor titles (names), but the variables are only populated in the text of Datadog child events (not the parent, which displays an aggregation summary). 
+
+* Some tags identifying your triggering scope are automatically inserted into the title of your multi alert. If your scope is defined by a lot of tags, your alert title may end up being undesirably long.
+
+#### {{comparator}} template variables
+
+The `{{comparator}}` template variable's value is always a relational operator. It correlates to the relational value selected in the monitor's "Set alert conditions" section:
+{{< img src="monitors/notifications/comparator_alert.png" alt="comparator_alert" responsive="true" popup="true" >}}
+
+For example, when an alert set to trigger when a value rises "above" 50, this syntax:
+```
+  {{value}} {{comparator}} {{threshold}}
+```
+would yield a notification message like the following:
+```
+    51.47 > 50
+```
+
+### Conditional variables
+Use conditional variables to display a different notification content depending of the context of your notification.
 
 The conditional variables available are:
 
-* `is_alert`, 
-* `is_alert_recovery`,
-* `is_warning`, 
-* `is_warning_recovery`, 
-* `is_recovery`, 
-* `is_no_data`.
+* `{{is_alert}}`, 
+* `{{is_alert_recovery}}`,
+* `{{is_warning}}`, 
+* `{{is_warning_recovery}}`, 
+* `{{is_recovery}}`, 
+* `{{is_no_data}}`,
+* `{{is_match}}`.
 
 These can also be seen in the "Use message template variables" help box in
 Step 3 of the monitor editor.
 
-### Tag variables for multi alerts
-When your monitor is a multi alert, instead of having a generic message (and finding the triggering tag scope in the alert query definition), a variable can be used in the message for explicitly identifying the triggering scope.
+#### `{{is_alert}}` & `{{is_warning}}`
 
-Here is an example of how you can use template variables for a multi alert:
+These variables use simple if-else logic to display a different message depending on whether the event is a trigger, warning, recovery, or no data notification with the following syntax:
```

Comment:
I was a bit confused reading this sentence. Could it be reworded as follows?

```
These variables use simple `if-else` logic to display a different message depending on the event type.
```

Reply:
👍